### PR TITLE
Update Shopify Littledata - Deploy

### PR DESCRIPTION
**Pull request by markzegarelli**
### Related issues (optional)
Deploy PR for #3539

### CODE REVIEW

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 1]
```diff
@@ -232,17 +232,17 @@ Each item in the `products` array, or Product Viewed and Product Added events, w
...
+With an [annual Littledata Plus plan](https://www.littledata.io/app/enterprise){:target="\_blank"} you can import all Shopify orders and refunds from before you started using Segment, to sync with destinations that support timestamped events (for example, a data warehouse). This enables you to build a complete customer history in your chosen destination.
 
 This data import will include all the [event properties](#event-properties) usually sent with an `Order Completed` event, including the [customer traits](#identify-calls).
 
 ## Advanced settings
 
-You can edit these data pipeline settings within Littledata's app.
+You can customize Littledata's Shopify source from the [data pipeline settings](https://help.littledata.io/posts/data-pipeline-settings/) in the Littledata admin. The general settings affect how we handle details such as orders, products and pageviews. The more advanced settings include:
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:241]: Does "we" refer to Segment or Littledata?

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md (hunk 1 above)
Comment (stayseesong) [reply src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:241]:
```suggestion
You can customize Littledata's Shopify source from the [data pipeline settings](https://help.littledata.io/posts/data-pipeline-settings/) in the Littledata admin. The general settings affect how Littledata handles details such as orders, products and pageviews. The more advanced settings include:
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 2]
```diff
@@ -4,15 +4,16 @@ redirect_from:
...
+Littledata's [Shopify to Segment connection](https://help.littledata.io/posts/segment-overview/){:target="\_blank"} uses a combination of client-side (browser) and server-side tracking to ensure 100% accurate data about your Shopify store in Segment. Littledata automatically integrates with Shopify and Shopify Plus sites to capture every customer touchpoint, including sales, marketing, customer and product performance data.
 
 Littledata is available as an independent [Shopify App](https://apps.shopify.com/segment-com-by-littledata).
 
 #### Client-side (device mode) tracking
 
-During the [installation process](https://blog.littledata.io/help/posts/segment-installation-guide/){:target="_blank"}, Littledata adds a `LittledataLayer.liquid` snippet to all pages (included in `theme.liquid`) on your Shopify store. The benefits of this approach include:
+During the [installation process](https://help.littledata.io/posts/segment-installation-guide/){:target="\_blank"}:
```
[the remark this answers is not in the document]
Comment (stayseesong) [reply src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:16]:
```suggestion
After the [installation process](https://help.littledata.io/posts/segment-installation-guide/){:target="\_blank"}:
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 3]
```diff
@@ -4,15 +4,16 @@ redirect_from:
...
 
 - Segment's Analytics.js 2.0 library is loaded on all pages, except for the checkout
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:18]:
```suggestion
- Segment's Analytics.js 2.0 library loads on all pages, except for the checkout.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 4]
```diff
@@ -4,15 +4,16 @@ redirect_from:
...
 - Includes a LittledataLayer data layer for all pages
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:19]:
```suggestion
- All pages Include a LittledataLayer data layer.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 5]
```diff
@@ -4,18 +4,19 @@ redirect_from:
...
 
 #### Client-side (device mode) tracking
 
-During the [installation process](https://blog.littledata.io/help/posts/segment-installation-guide/){:target="_blank"}, Littledata adds a `LittledataLayer.liquid` snippet to all pages (included in `theme.liquid`) on your Shopify store. The benefits of this approach include:
+After the [installation process](https://help.littledata.io/posts/segment-installation-guide/){:target="\_blank"}:
 
-- Segment's Analytics.js 2.0 library is loaded on all pages, except for the checkout
-- Includes a LittledataLayer data layer for all pages
+- Segment's Analytics.js 2.0 library loads on all pages, except for the checkout.
+- All pages Include a LittledataLayer data layer.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:19]:
```suggestion
- All pages include a LittledataLayer data layer.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 6]
```diff
@@ -40,20 +41,20 @@ Here's an architecture diagram that shows how the Littledata app mediates data f
 ## Getting Started
 
 1. **Login** to your Shopify Store account.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:43]:
```suggestion
1. **Log in** to your Shopify Store account.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 7]
```diff
@@ -40,20 +41,20 @@ Here's an architecture diagram that shows how the Littledata app mediates data f
...
    ![](images/Nd5L0C6.png)
 3. Click **Add app** to begin the installation process.
 4. **Choose a Littledata subscription** suitable for your store's volume of monthly orders.
 5. Add the [**Segment write key**](/docs/connections/find-writekey/) for the source that is going to send data in the **input field**.
    ![](images/eLUh6GF.png)
-6. Choose either an **Automatic**, a **Manual**, or a **Headless** install. _Automatic installs work in most instances, but if you choose to do a manual install, just follow [this guide](https://blog.littledata.io/help/posts/segment-installation-guide/){:target="_blank"}._
+6. Choose either an **Automatic**, a **Manual**, or a **Headless** install. _Automatic installs work in most instances, but if you choose to do a manual install, just follow [this guide](https://help.littledata.io/posts/segment-installation-guide/){:target="\_blank"}._
    ![](images/iYM76VI.png)
 7. Segment's **Analytics.js** library, Littledata **tracking script** and **webhooks** will be automatically applied to the store and the installation process will then be complete.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:52]:
```suggestion
7. Segment's **Analytics.js** library, Littledata **tracking script** and **webhooks** automatically apply to the store and the installation process will then be complete.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 8]
```diff
@@ -232,17 +232,17 @@ Each item in the `products` array, or Product Viewed and Product Added events, w
 
 ## Import all orders
 
-With an [annual Littledata Plus plan](https://www.littledata.io/app/enterprise){:target="_blank"} you can import all Shopify orders and refunds from before you started using Segment, to sync with destinations that support timestamped events (for example, a data warehouse). This enables you to build a complete customer history in your chosen destination.
+With an [annual Littledata Plus plan](https://www.littledata.io/app/enterprise){:target="\_blank"} you can import all Shopify orders and refunds from before you started using Segment, to sync with destinations that support timestamped events (for example, a data warehouse). This enables you to build a complete customer history in your chosen destination.
 
 This data import will include all the [event properties](#event-properties) usually sent with an `Order Completed` event, including the [customer traits](#identify-calls).
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:237]:
```suggestion
This data import includse all the [event properties](#event-properties) usually sent with an `Order Completed` event, including the [customer traits](#identify-calls).
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 9]
```diff
@@ -4,18 +4,19 @@ redirect_from:
...
 id: V8ji9rWzoS
 ---
+
 <!-- LR Note: the working copy of the source catalog YML we built on showed this in the `website` source though as of Nov 18 it's labeled cloud-source -->
 
-Littledata's [Shopify to Segment connection](https://blog.littledata.io/help/posts/segment-overview/){:target="_blank"} uses a combination of client-side (browser) and server-side tracking to ensure 100% accurate data about your Shopify store in Segment. Littledata automatically integrates with Shopify and Shopify Plus sites to capture every customer touchpoint, including sales, marketing, customer and product performance data.
+Littledata's [Shopify to Segment connection](https://help.littledata.io/posts/segment-overview/){:target="\_blank"} uses a combination of client-side (browser) and server-side tracking to ensure 100% accurate data about your Shopify store in Segment. Littledata automatically integrates with Shopify and Shopify Plus sites to capture every customer touchpoint, including sales, marketing, customer and product performance data.
 
 Littledata is available as an independent [Shopify App](https://apps.shopify.com/segment-com-by-littledata).
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:12]:
```suggestion
Littledata is available as an independent [Shopify App](https://apps.shopify.com/segment-com-by-littledata){:target="\_blank"}.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md (hunk 8 above)
Comment (edwardupton) [reply src/connections/sources/catalog/libraries/website/shopify-littledata/index.md]: typo in `includse`

**Comment by edwardupton** [comment]
Just one typo fix and then looks good to me

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 10]
```diff
@@ -4,18 +4,19 @@ redirect_from:
...
-During the [installation process](https://blog.littledata.io/help/posts/segment-installation-guide/){:target="_blank"}, Littledata adds a `LittledataLayer.liquid` snippet to all pages (included in `theme.liquid`) on your Shopify store. The benefits of this approach include:
+After the [installation process](https://help.littledata.io/posts/segment-installation-guide/){:target="\_blank"}:
 
-- Segment's Analytics.js 2.0 library is loaded on all pages, except for the checkout
-- Includes a LittledataLayer data layer for all pages
+- Segment's Analytics.js 2.0 library loads on all pages, except for the checkout.
+- All pages include a LittledataLayer data layer.
 - Loads a minified tracking script, hosted on a content delivery network (CDN)
 - Enables sending of device-mode ecommerce events to all Segment destinations
 - Segment's anonymous ID and Google Analytics' client ID is passed to our servers to ensure consistent user journey tracking
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:22]:
```suggestion
- Pages Load a minified tracking script, hosted on a content delivery network (CDN)
- Device-mode e-commerce events can send to all Segment destinations
- Segment's anonymous ID and Google Analytics' client ID passes to Littledata's servers to ensure consistent user journey tracking
```

**Review by stayseesong — APPROVED** [review]

**Review by stayseesong — APPROVED** [review]

**Pull request merged**